src/libs/ReportActionsUtils.ts

```diff
@@ -5,21 +5,37 @@ import OnyxUtils from 'react-native-onyx/lib/utils';
 import {ValueOf} from 'type-fest';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
-import {ActionName} from '@src/types/onyx/OriginalMessage';
+import {ActionName, ChangeLog} from '@src/types/onyx/OriginalMessage';
 import Report from '@src/types/onyx/Report';
-import ReportAction, {ReportActions} from '@src/types/onyx/ReportAction';
+import ReportAction, {Message, ReportActions} from '@src/types/onyx/ReportAction';
 import {EmptyObject, isEmptyObject} from '@src/types/utils/EmptyObject';
 import * as CollectionUtils from './CollectionUtils';
 import * as Environment from './Environment/Environment';
 import isReportMessageAttachment from './isReportMessageAttachment';
+import * as Localize from './Localize';
 import Log from './Log';
+import {MessageElementBase, MessageTextElement} from './MessageElement';
+import * as PersonalDetailsUtils from './PersonalDetailsUtils';
```

Comment:
This caused regression of dependency cycle

![console](https://github.com/Expensify/App/assets/108292595/ba9f7e1d-35be-48a3-8db7-f2e6c9e812b6)

Reply:
I see `en.ts` import `ReportActionsUtils` to use here
https://github.com/Expensify/App/blob/2feaac80198481546ee6aa630aa1c7b386ec5605/src/languages/en.ts#L428-L430

And in the `ReportActionsUtils.ts` we import `Localize.ts`
=> Resolve: we can in `en.ts` we can remove `ReportActionsUtils` and use logic check like this
```js
editAction: ({action}: EditActionParams) => `Edit ${action?.actionName === CONST.REPORT.ACTIONS.TYPE.IOU ? 'request' : 'comment'}`,
```
What do you think?

Reply:
Agree. I also don't like importing any utils in language file